### DISCUSSION TITLE
使用ch_PP-OCRv4_det_server_train模型训练数据卡住

### DISCUSSION THREAD

**Discussion by ying-jie-li** [PaddlePaddle/PaddleOCR#12879] [Q&A]
请提供下述完整信息以便快速定位问题

- 系统环境/System Environment：ubuntu 18.04       cuda11.2 python3.8  
- 版本号/Version：Paddle：2.5.1gpu   PaddleOCR：Name: paddleocr
Version: 2.7.0.3
 问题相关组件/Related components：paddleOCR 
- 运行指令/Command Code： CUDA_VISIBLE_DEVICES=0 python tools/train.py -c /software/newPaddleOCR/PaddleOCR/configs/det/ch_PP-OCRv4/ch_PP-OCRv4_det_student.yml

- 完整报错/Complete Error Message：
- 没有报错，但是界面一直卡住，前面也有我提出的问题， 就是会卡住，自己用源码进行训练的时候
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/54535654/0c1b7f56-e0a7-4728-9a4c-b424b11f0aba)

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by data-ant** [comment]
解决了吗？
我也遇到了卡住的问题，谢谢
